src/view/add-and-edit-event-form-view.js

```diff
@@ -28,44 +40,85 @@ function createEventTypeItemTemplate(eventTypesList, selectedEeventType) {
 
 function createOfferTemplate(offers, selectedOffers) {
   let offerTemplate = '';
-  if (offers && offers.length !== 0) {
-    const selectedOffersSet = new Set(selectedOffers.map((offer) => offer.id));
-
-    for (let i = 0; i < offers.length; i++) {
-      const selectedOfferAttribute = selectedOffersSet.has(offers[i].id) ? 'checked' : '';
-      offerTemplate += `
-        <div class="event__offer-selector">
-          <input class="event__offer-checkbox  visually-hidden" id="event-offer-offer${i}-1" type="checkbox" name="event-offer-offer${i}" ${selectedOfferAttribute}>
-          <label class="event__offer-label" for="event-offer-offer${i}-1">
-            <span class="event__offer-title">${offers[i].title}</span>
-            &plus;&euro;&nbsp;
-            <span class="event__offer-price">${offers[i].price}</span>
-          </label>
-        </div>
+  const selectedOffersSet = new Set(selectedOffers.map((offer) => offer.id));
+
+  for (let i = 0; i < offers.length; i++) {
+    const selectedOfferAttribute = selectedOffersSet.has(offers[i].id) ? 'checked' : '';
+    offerTemplate += `
+      <div class="event__offer-selector">
+        <input class="event__offer-checkbox  visually-hidden" id="event-offer-offer${i}-1" type="checkbox" name="event-offer-offer${i}" ${selectedOfferAttribute}>
+        <label class="event__offer-label" for="event-offer-offer${i}-1">
+          <span class="event__offer-title">${offers[i].title}</span>
+          &plus;&euro;&nbsp;
+          <span class="event__offer-price">${offers[i].price}</span>
+        </label>
+      </div>
       `;
-    }
   }
   return offerTemplate;
 }
 
+function createOffersBlockTemplate(offers, selectedOffers) {
+  let offersBlockTemplate = '';
+  if (offers && offers.length !== 0) {
+    offersBlockTemplate += `
+    <section class="event__section  event__section--offers">
+      <h3 class="event__section-title  event__section-title--offers">Offers</h3>
+
+      <div class="event__available-offers">
+        ${createOfferTemplate(offers, selectedOffers)}
+      </div>
+    </section>
+    `;
+  }
+  return offersBlockTemplate;
+}
+
+function createDestinationBlockTemplate(cityName, cityDescription, isEditEventForm, pictures) {
+  let destinationBlockTemplate = '';
+  if (cityName && cityName !== '') {
```

Comment:
Если `cityName` будет равен `''`, то первая часть проверки уже упадёт, поэтому нет смысла отдельно проверять на это значение

```suggestion
if (cityName) {
```